numba_dppy/__init__.py

```diff
@@ -208,10 +208,10 @@ def main():
         b = np.array(np.random.random(N), dtype=np.float32)
         c = np.ones_like(a)
 
-        if dpctl.has_gpu_queues():
+        if dpctl.has_opencl_gpu():
```

Comment:
Dose `dpctl` have `has_opencl_gpu` ?

Reply:
No. What is the intent behind these changes? The whole docstring needs to be updated. It is on my long list of todo

Reply:
Yes, you're right, there is no such method. In general, this example itself is outdated, because the method `has_gpu_queues` no longer exists.